src/common/Error/styled.js

```diff
@@ -0,0 +1,93 @@
+import styled from "styled-components";
+import { ReactComponent as Error } from "./error.svg";
+
+export const Wrapper = styled.div`
+  margin: 195px auto 0 auto;
+  text-align: center;
+  height: 100vh;
+  
+  @media (max-width: ${({ theme }) => theme.breakpoint.small}) {
+    margin-top: 100px;
+  }
+`;
+
+export const ErrorIcon = styled(Error)`
+  width: 100px;
+  transition: 0.3s;
+
+  @media (max-width: ${({ theme }) => theme.breakpoint.small}) {
+    width: 80px;
+  }
+  
+  .animate {
+    animation: wobble 1s ease-in-out infinite;
```

Comment:
COOL animation dude! I'm impressed 😎

P.S. I think this `.animate {` class wrap is unnecessary.


`};`